src/components/Subtitle.tsx

```diff
@@ -1,4 +1,13 @@
 import React from 'react'
```

Comment:
Kann man das ohne diesen Import nutzen? Falls nicht, dann müssen wir noch was an unserem TS/Webpack setup anpassen?!

Reply:
Das geht irgendwie, siehe https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html
Aber ich möchte jetzt erst mal nicht noch babel-support zum Projekt hinzufügen aktuell. Lass uns das in einer dedizierten Story machen!